src/translations/en.json

```diff
@@ -2220,6 +2220,28 @@
                 }
               }
             }
+          },
+          "agents": {
+            "cloud_agent_description": "It stores one backup. The oldest backups are deleted.",
+            "no_agents": "No sync agents configured"
+          },
+          "data": {
+            "ha_settings_included": "Home Assistant settings are always included",
+            "ha_settings": "Home Assistant settings",
+            "ha_settings_description": "The bare minimum needed to restore your system.",
+            "history": "History",
+            "history_description": "Historical data of your sensors, including your energy dashboard.",
+            "media": "Media",
+            "media_description": "For example, camera recordings.",
+            "share_folder": "Share folder",
+            "share_folder_description": "Folder that is often used for advanced or older configurations.",
+            "local_addons": "Local addons folder",
```

Comment:
Shouldn't that be "Local add-ons folder"?

You used the hyphen in all other strings.